src/components/reusable/MyTable.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from "react";
+import moment from "moment";
+import PropTypes from "prop-types";
+
+const MyTable = ({ columns, data }) => {
+  const [tableColumns, setTableColumns] = useState([]);
+  const [tableData, setTableData] = useState([]);
+
+  useEffect(() => {
+    setTableData(data);
+    setTableColumns(columns);
+  }, []);
+
+  const rowClick = selectedItem => {
+    const newTableData = tableData.map(item => {
+      if (selectedItem.key === item.key) {
+        return {
+          ...item,
+          showContent: !item.showContent
+        };
+      }
+      return item;
+    });
+    setTableData(newTableData);
+  };
+
+  const headClick = sortKey => {
+    const sortedData = [...tableData];
+
+    if (typeof data[0][sortKey] === "number") {
+      sortedData.sort((a, b) => a[sortKey] - b[sortKey]);
```

Comment:
Tip: I would prefer to use something like `lodash.orderBy`, it has nice API it's lightweight and you don't have to create a shallow copy of your array.

Reply:
DONE